op-challenger/config/config_test.go

```diff
@@ -24,6 +24,14 @@ var (
 	validRollupRpc             = "http://localhost:8555"
 )
 
+var (
```

Comment:
nit: merge these blocks. Can delineate with an empty line between for legibility.

```suggestion

```